metta/metta_io.py

```diff
@@ -21,7 +21,8 @@ def archive_train_test(train_config,
                        df_test,
                        directory='.',
                        format='hd5',
-                       overwrite=False):
+                       overwrite=False,
+                       write_index=True):
```

Comment:
Should the default behavior be False? Most of the time we wouldn't want pandas to write an index?